video4.html

```diff
@@ -1,4 +1,4 @@
-<a href='/'><button>Voltar</button></a>
+<a href='/cursos/curso1'><button>Voltar</button></a>
 <p>Node.js API testável #4 - Models com Sequelize</p>
 <iframe width="560" height="315" src="https://www.youtube.com/embed/J3IQVGSMkLI" frameborder="0" allowfullscreen></iframe>
 <br><br>
```

Comment:
A tag "<br">" não deve ser utilizada para separar elementos.

Mais sobre isso: https://www.tipsandtricks-hq.com/10-common-html-mistakes-to-avoid-1980